### ISSUE TITLE
[a11y]: The 'id' "downshift-:r2:-label" specified for the ARIA property 'aria-labelledby' value is not valid

### ISSUE THREAD

**Issue by parul-26** [carbon-design-system/carbon#18858]
### Package

@carbon/react

### Browser

Chrome

### Operating System

MacOS

### Package version

v1.40.0

### Automated testing tool and ruleset

IBM Equal Access Accessibility Checker - Latest Deployment

### Description

The 'id' "downshift-:r2:-label" specified for the ARIA property 'aria-labelledby' value is not valid

<img width="1726" alt="Image" src="https://github.com/user-attachments/assets/8c01e7de-c0b8-498b-9bf7-12fa7ece582b" />

### Reproduction/example

https://stackblitz.com/edit/github-cmfcow-9vyb8z?file=src%2FApp.jsx

### Steps to reproduce

For the Dropdown of the file i got this Violation
The 'id' "downshift-:r2:-label" specified for the ARIA property 'aria-labelledby' value is not valid.

### Suggested Severity

Severity 2 = User cannot complete task, and/or no workaround within the user experience of a given component.

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by parul-26** [comment]
https://github.com/carbon-design-system/carbon/issues/17567 is the reference issue. we are able to see the voilations again for the Dropdown component.

**Comment by parul-26** [comment]
hey @2nikhiltom , I have found that if we are using the Tooltip component inside the Dropdown, it causes a violation.
here is the [stackblitz](https://stackblitz.com/edit/github-jdzenu?file=src%2FApp.jsx). can you please reopen this issue.

**Comment by tay1orjones** [comment]
@parul-26 could you please add that info to https://github.com/carbon-design-system/carbon/issues/17582?
